### ISSUE TITLE
cmd/go: -keepfuzzing needs renaming, does not exist

### ISSUE THREAD

**Issue by rsc** [golang/go#48127]
Generally, we try to keep the flag names big-endian, so that related flags all have a common prefix, like bench, benchtime, benchmem; memprofile, memprofilerate; even fuzz, fuzzcachedir, fuzzminimizetime, fuzztime, fuzzworker. 

The exception in the go help testflag output is keepfuzzing, which doesn't begin with fuzz. It probably should. Maybe fuzzmore?

Also the flag seems not to be implemented. 

	% gotip test -v -fuzz=Fuzz -keepfuzzing
	flag provided but not defined: -keepfuzzing
	Usage of /tmp/go-build1779596492/b001/pkg.test:
		...

/cc @jayconrod @katiehockman

**Comment by ameowlia** [comment]
fuzzforever? fuzzon? fuzznonstop?

**Comment by rsc** [comment]
fuzzcontinue would probably be fine.

**Comment by katiehockman** [comment]
https://go-review.googlesource.com/c/go/+/351113 removed the documentation for `-keepfuzzing` since it likely won't make it into this release. (And can always be added back if it does for whatever reason)
So I'll remove the release-blocker label.

`-fuzzcontinue` seems reasonable to me for when we implement it.

**Comment by rogpeppe** [comment]
FWIW I'd expect this to be the default - if I'm leaving something fuzzing for days or weeks (as I'm doing currently), I don't want it to stop immediately it finds a crash.
I was surprised when I found out that it did.

Possible flag name: `-fuzzonce` with the opposite sense to `-fuzzcontinue`? `-fuzzstoponfail` ?

**Comment by cespare** [comment]
@rogpeppe or use the existing `-failfast` flag?

(Though I'm not sure I agree about inverting the default behavior.)

**Comment by katiehockman** [comment]
This won't happen for 1.18, but I'd like to see it in 1.19. Updated the milestone.

**Comment by ianlancetaylor** [comment]
CC @golang/fuzzing 

Do we want this?  The milestone is currently 1.19.  Should it be 1.20?  Backlog?  Thanks.

**Comment by matloob** [comment]
We don't have plans to implement this at the moment.

**Comment by mvdan** [comment]
@matloob could you expand on the reasoning a bit? It seems like a handful of people are surprised by the decision.

**Comment by mknyszek** [comment]
I'm going to guess (and @matloob can clarify) that:

1. This issue is about naming a flag that doesn't actually exist, so it doesn't make sense to keep it open.
2. We don't plan on implementing the flag in the near future.

That being said, we should probably have a separate issue and/or proposal for adding the flag, since there's clearly interest.

**Comment by jakebailey** [comment]
Yes, I can imagine a number of names that'd work for (1), but I was more confused at (2). It's very frustrating to hit a failing fuzz test that you can't fix yet (or, aren't fixing becuase this is a bulk CI operation or something), and just not be able to look for more crashers.
